public/data/maps/plantLocations.json

```diff
@@ -0,0 +1,898 @@
+{
+  "id": "proj_WZkyugryh35sMmZMmXCwq7YY",
```

Comment:
If this file contains some demo data, it should be removed before merging this pull request.